# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/serviceworkers/moz.build
# Commit: 679bed36bc0b
# Full Hash: 679bed36bc0b4dbca32def451b6385e5a898368b
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2021-06-10 09:49:36
# Regressor Bug: 1564235
# File Overlap Count: 2
# Description:
#   Bug 1564235 - P3 Create IPC for NavigationPreload in PServiceWorkerRegistration.ipdl. r=dom-worker-reviewers,asuth
#   
#   This patch implements the IPC for NavigationPreload APIs.
#   The IPC methods are created on PServiceWorkerRegistration.
#   
# ==============================================================================

diff -r 789ce1a19e9f -r 679bed36bc0b dom/serviceworkers/moz.build
--- a/dom/serviceworkers/moz.build	Thu Jun 10 00:48:11 2021 +0000
+++ b/dom/serviceworkers/moz.build	Thu Jun 10 00:48:11 2021 +0000
@@ -90,6 +90,7 @@
 ]
 
 IPDL_SOURCES += [
+    "IPCNavigationPreloadState.ipdlh",
     "IPCServiceWorkerDescriptor.ipdlh",
     "IPCServiceWorkerRegistrationDescriptor.ipdlh",
     "PFetchEventOp.ipdl",
